tests/Unit/PointwiseFunctions/GeneralRelativity/Test_ComputeGhQuantities.cpp

```diff
@@ -128,6 +141,68 @@ void test_lapse_deriv_functions(const DataVector& used_for_size) noexcept {
       {{{std::numeric_limits<double>::denorm_min(), 10.}}}, used_for_size);
 }
 
+// Test computation of derivs of lapse by comparing to Kerr-Schild
+template <typename Solution>
+void test_lapse_deriv_functions_analytic(
+    const Solution& solution, const size_t grid_size_each_dimension,
+    const std::array<double, 3>& lower_bound,
+    const std::array<double, 3>& upper_bound) noexcept {
+  // Setup grid
+  const size_t SpatialDim = 3;
```

Comment:
`spatial_dim` (we use snake_case for variables and CamelCase for all template parameters, just so you know why it's CamelCase in a lot of places :) )

Reply:
oops, fixed now